<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RowDacTester</class>
 <widget class="QDialog" name="RowDacTester">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>309</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Row and DAC tester</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>*NOTE1: Normal keyboard input is disabled while this window is open.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>*NOTE2: Please select the DAC threshold to output.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="second_label_2">
     <property name="text">
      <string>*NOTE3: The squares below the DAC settings show the status of each row.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="second_label_3">
     <property name="text">
      <string>*NOTE4: Rows are in the order seen by software.</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="last_label">
       <property name="text">
        <string>*NOTE3: Colors:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="last_label_2">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #008000; color: #FFFFFF; border: 1px solid black;</string>
       </property>
       <property name="text">
        <string>LOW</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="last_label_3">
       <property name="styleSheet">
        <string notr="true">background-color: #00FF00; color: #000000;border: 1px solid black;</string>
       </property>
       <property name="text">
        <string>HIGH</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Supply voltage:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>DAC output voltage:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QSlider" name="horizontalSliderSupply">
         <property name="maximum">
          <number>600</number>
         </property>
         <property name="value">
          <number>500</number>
         </property>
         <property name="sliderPosition">
          <number>500</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Threshold voltage:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSlider" name="horizontalSlider">
         <property name="maximum">
          <number>1023</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Threshold value:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="spinBox">
       <property name="maximum">
        <number>1023</number>
       </property>
       <property name="value">
        <number>1023</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxSupply">
       <property name="suffix">
        <string> V</string>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>5.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLabel" name="dac_voltage">
       <property name="styleSheet">
        <string notr="true">background-color: #ffffff; color: #000000; border: 1px solid black</string>
       </property>
       <property name="text">
        <string>STARTING...</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QLabel" name="threshold_voltage">
       <property name="styleSheet">
        <string notr="true">background-color: #ffffff; color: #000000; border: 1px solid black;</string>
       </property>
       <property name="text">
        <string>STARTING...</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>3</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
